src/utils/constants/routes.ts

```diff
@@ -1,4 +1,4 @@
-export const ROUTES = {
+export const ROUTES: Record<string, string> = {
```

Comment:
Redundant; typescript will inherit the types automatically.

Reply:
When removing this part of code, I encounter an error.